### PR TITLE
Add snapshot tests

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add snapshot tests for `MediaChip`, `SeekBackButton` and `SeekForwardButton`.

#### WHY

Suits better than the current unit tests.
Also some of the unit tests are ignored due to flakiness.

#### HOW

- Add snapshot tests for variants of the components;
- Add A11y snapshot tests;

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

**Comment by yschimke** [comment]
It is possible to use this without full screen images. My be a nice improvement before we use more widely.

Setup the Paparazzi with a device config a minimum width and height of 32px
Set renderingMode to FULL_EXPAND

Whatever you render inside paparazzi.snapshot should expand to be it's "natural" size.

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt [hunk 1]
```diff
@@ -61,8 +61,8 @@ fun UampEntityScreen(
 ) {
     val uiState by StateUtils.rememberStateWithLifecycle(flow = uampEntityScreenViewModel.uiState)
 
-    var showCancelDownloadsDialog by remember { mutableStateOf(false) }
-    var showRemoveDownloadsDialog by remember { mutableStateOf(false) }
+    var showCancelDownloadsDialog by rememberSaveable { mutableStateOf(false) }
```
Comment (yschimke) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt:64]: Nice, I don't typically think of this. Wear Compose takes care of it for Scrolling etc.

Good example.

File: media-ui/src/main/java/com/google/android/horologist/media/ui/util/Talkback.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.horologist.media.ui.util
+
+/**
+ * Make explicit that a conscious decision was made to hide the content description from talkback.
+ */
+internal val CONTENT_DESCRIPTION_HIDDEN_FROM_TALKBACK: String? = null
```
Comment (yschimke) [inline media-ui/src/main/java/com/google/android/horologist/media/ui/util/Talkback.kt:22]: Nice. Better than the comment.

**Review by yschimke — APPROVED** [review]

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt (hunk 1 above)
Comment (yschimke) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt:64]: Are these specifically related to the PR?

**Comment by yschimke** [comment]
This looks great.  Nice to know from the bottom up that every composable is designed for A11y.

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt (hunk 1 above)
Comment (luizgrp) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt:64]: no, they are unrelated. can move to another pr if you'd prefer?

Comment (yschimke) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt:64]: Your call.

Comment (luizgrp) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/entity/UampEntityScreen.kt:64]: https://github.com/google/horologist/pull/650

**Pull request merged**